tensorpack/dataflow/common.py

```diff
@@ -149,6 +149,60 @@ def _aggregate_batch(data_holder, use_list=False):
         return result
 
 
+class BatchDataByMixup(BatchData):
```

Comment:
Could you put this in the example script? And it would seem easier to write it as a `MapData` on top of a batch.

Reply:
yes, i can do that, but in that way I still need hack the MapData method in the "common.py", because mixup needs two data, however, the MapData only returns one. any good suggestion?

Reply:
What you've written is literally `MapData(BatchData(ds, batch_size), mixup)`.